GYBITG/GalleryViewController.swift

```diff
@@ -14,8 +14,15 @@ import UIKit
 import MobileCoreServices
 import CoreMedia
 
+//protocol for all repos to inherit from so polymorphism can be used for the UIStoryboardSegue
+//Each  Repository will define and return which type of Repo they are
+protocol Repo{
+    func type()-> Character
```

Comment:
This is a good idea, my only critique is maybe changing the returning data type of the 'type()' function to a String rather than a Character, I think this would make it a little more clear as to what is going on (especially to an outsider) in the UIStoryboardSegue. Also, maybe move the Repo protocol out of the GalleryViewController and into the UIStoryboardSegue since that is where it is used. Other than that, nice job!

Reply:
Good suggestions, I will make those changes. Thanks!